.github/workflows/ci.yml

```
@@ -1,362 +1,361 @@
+                        tests/vyper/ethereum/memoryManagement, # PASSING
+                        tests/vyper/ethereum/operators, # PASSING
+                        tests/vyper/ethereum/optimizer, # PASSING
+                        tests/vyper/ethereum/revertStrings, # PASSING
+                        tests/vyper/ethereum/reverts, # FAILING
+                        tests/vyper/ethereum/smoke, # PASSING
+                        tests/vyper/ethereum/specialFunctions, # PASSING
+                        tests/vyper/ethereum/state, # PASSING
+                        tests/vyper/ethereum/storage, # PASSING
+                        tests/vyper/ethereum/strings, # PASSING
+                        tests/vyper/ethereum/structs, # PASSING
+                        tests/vyper/ethereum/types, # PASSING
+                        tests/vyper/ethereum/underscore, # PASSING
+                        tests/vyper/ethereum/variables, # PASSING
+                        tests/vyper/ethereum/various, # PASSING
+                        tests/vyper/ethereum/viaYul, # PASSING
+                        tests/vyper/simple/*.vy, # PASSING
+                        tests/vyper/simple/algorithm, # PASSING
+                        tests/vyper/simple/array, # PASSING
+                        tests/vyper/simple/block, # PASSING
+                        tests/vyper/simple/built_in_functions, # PASSING
+                        tests/vyper/simple/conditional, # PASSING
+                        tests/vyper/simple/constructor, # PASSING
+                        tests/vyper/simple/destructuring, # PASSING
+                        tests/vyper/simple/error, # PASSING
+                        tests/vyper/simple/events, # PASSING
+                        tests/vyper/simple/expression, # PASSING
+                        tests/vyper/simple/fallback, # PASSING
+                        tests/vyper/simple/function, # PASSING
+                        tests/vyper/simple/immutable, # PASSING
+                        tests/vyper/simple/interface, # PASSING
+                        tests/vyper/simple/loop, # PASSING
+                        tests/vyper/simple/modular, # PASSING
+                        tests/vyper/simple/operator, # PASSING
+                        tests/vyper/simple/order, # PASSING
+                        tests/vyper/simple/overflow, # PASSING
+                        tests/vyper/simple/return, # PASSING
+                        tests/vyper/simple/revert_on_failure, # PASSING
+                        tests/vyper/simple/solidity_by_example, # PASSING
+                        tests/vyper/simple/storage, # PASSING
+                        tests/vyper/simple/structure, # PASSING
+                        tests/vyper/simple/unchecked_math, # PASSING
+                        tests/vyper/simple/unused, # PASSING
+                        "tests/yul/{*.yul,examples,instrucions,near_call_abi,precompiles,semantic,simulations}", # PASSING
```

Since this includes all `tests/yul` tests, you may simplify it to:
```suggestion
                        tests/yul, # PASSING
```